test/react-native/features/app_data.feature

```diff
@@ -0,0 +1,81 @@
+Feature: App data
+
+Scenario: Handled JS error
+  When I run "AppJsHandledScenario" and relaunch the app
+  Then I wait to receive a request
+  And the exception "errorClass" equals "Error"
+  And the exception "message" equals "HandledCaughtError"
+  And the event "app.version" equals "1.2.3"
+  And the event "app.versionCode" equals 1
+  And the event "app.releaseStage" equals "production"
+  And the event "app.inForeground" is true
+  And the event "app.duration" is not null
+  And the event "app.durationInForeground" is not null
+
+  # Android
+  And the event "app.id" equals "com.reactnative"
+  And the event "app.type" equals "android"
+  # iOS
+  # And the event "app.id" equals "org.reactjs.native.example.reactnative"
+  # And the event "app.type" equals "ios"
+
+
+Scenario: Unhandled JS error
+  When I run "AppJsUnhandledScenario" and relaunch the app
+  And I configure Bugsnag for "AppJsUnhandledScenario"
+  Then I wait to receive a request
+  And the exception "errorClass" equals "Error"
+  And the exception "message" equals "UnhandledError"
+  And the event "app.version" equals "1.2.3"
+  And the event "app.versionCode" equals 1
+  And the event "app.releaseStage" equals "production"
+  And the event "app.inForeground" is true
+  And the event "app.duration" is not null
+  And the event "app.durationInForeground" is not null
+
+  # Android
+  And the event "app.id" equals "com.reactnative"
```

Comment:
You've probably seen already that this fails on v0.55 - possibly an issue for the fixture?

Reply:
Not sure how to resolve. @Cawllec could we get the package name to be `com.reactnative` on both versions? Or do we need to match against multiple strings here.

Reply:
If the answer to this isn't simple and in the interest of getting the PR merged, I would be happy to see this check commented out in the short term, with a reference to an internal ticket to be actioned before release.